cypress/e2e/e2e.cy.js

```diff
@@ -398,6 +398,48 @@ describe("Mobile Flow", () => {
     })
 })
 
+describe("Test Offline Mode with Navigation", () => {
+    it("Journal Offline Mode and Upload When Back Online", () => {
+        let onlineStub;
+
+        Cypress.on('window:before:load', (win) => {
+            onlineStub = cy.stub(win.navigator, 'onLine').value(false);
+        });
+
+        cy.intercept("POST", "/moodLog", {
+            forceNetworkError: true,
+        }).as("postJournal");
+
+        cy.visit("/journal");
+
+        cy.contains("What's happening").should("exist");
+        cy.get("textarea").should("exist").focus();
+        cy.get("textarea").type("This is an offline test");
+        cy.contains("Continue").should("exist").click();
+        cy.contains("Done!").should("exist").click();
+
+        cy.url().should("include", "/summary");
+        cy.get(".mood-card-log").last().contains("This is an offline test");
+        cy.get(".mood-card").last().find("#cloudOffline").should("exist");
+
+        cy.intercept("POST", "/moodLog", (req) => {
+            req.continue();
+        });
+        cy.then(() => {
+            onlineStub.value(true);
+            cy.window().then((win) => {
+                win.dispatchEvent(new Event('online'));
+            });
+        });
+
+        cy.wait(WAIT_FOR_CONSISTENCY)
```

Comment:
is this wait required? could we just increase the timeout on #cloudOffline should not exist